docs/topic2.md

```diff
@@ -1,3 +1,115 @@
-# Topic 2 - CSS & HTML
+# Topic 2 - CSS
+
+[-> To index](../README.md#title)
```

Comment:
Please remove the link to index from the beginning of the readme, and read the link to Index and next topic at the end